src/vmeasure.jl

```diff
@@ -0,0 +1,65 @@
+# V-measure of contingency table
+function _vmeasure(A::Matrix{Int}; β = 1.0)
+    @assert β >= 0 "β should be nonnegative"
+
+    C, K = size(A)
+    N = sum(A)
+
+    # Homogeneity
+    hck = 0.0
+    for k in 1:K
+        Ak = view(A, :, k)
+        d = sum(Ak)
+        for c in 1:C
+            if Ak[c] != 0 && d != 0
+                hck += log(Ak[c]/d) * Ak[c]/N
```

Comment:
> Because `d` value depends on a particular column, it would require to run loop over column (row) to calculate such scaling factor, `∑Ak/N log(N/∑Ac)`.

Sure, but you can still apply `entropy()` column-wise:
```julia
hkc += entropy(Ak)/N + log(d)*d/N
```
Also you can check `d != 0` before the `c` loop.

Actually, I think we can avoid the loops completely and just use `entropy()` for everything. This seem to pass the tests:
```julia
    N = sum(A)
    (N == 0.0) && return 0.0

    entA = entropy(A)
    entArows = entropy(sum(A, 2))
    entAcols = entropy(sum(A, 1))

    hck = (entA - entAcols)/N
    hkc = (entA - entArows)/N
    hc = entArows/N + log(N)
    hk = entAcols/N + log(N)

    # Homogeneity
    h = hc == 0.0 ? 1.0 : 1.0 - hck/hc
    # Completeness
    c = hk == 0.0 ? 1.0 : 1.0 - hkc/hk
```
It avoids reevaluating `log(A[i,j])` and column/row-wise sums (`d`).